public/data/Abdus_Salam.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Abdus Salam",
+  "type": "personal",
+  "bio": "Hi there! I am Python Developer and doing my best to contribute to open source. I'm not limited to Python only, I can do more fun things! (it's a long list) Connect with me and find out",
+  "avatar": "https://github.com/abdussalam02.png",
+  "links": [
+    {
+        "name": "Check out my personal website",
+        "url": "https://abdussalam.vercel.app",
+        "icon": "globe"
+    },
+    {
+      "name": "Take a look at my GitHub",
+      "url": "https://github.com/abdussalam02",
+      "icon": "github"
+    },
+    {
+        "name": "I'm on Telegram most of my time",
+        "url": "t.me/aladdin_02",
+        "icon": "telegram"
+    },
+    {
+      "name": "See my awesome tweets on Twitter",
+      "url": "https://twitter.com/abdussalam02",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://linkedin.com/in/",
```

Comment:
Your LinkedIn url is missing your username

Reply:
sorry i forgot